server/service.go

```diff
@@ -338,16 +343,15 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 		"msIntoSlot":  msIntoSlot,
 	}).Infof("getHeader request start - %d milliseconds into slot %d", msIntoSlot, _slot)
 
-	// Add request headers
```

Comment:
FYI-I had to remove 3 commented line to make linter happy. Otherwise `handleGetHeader` method having below linter issue.
` Function name: handleGetHeader, Cyclomatic Complexity: 24, Halstead Volume: 6038.34, Maintainability Index: 19`